### PR TITLE
Update to Java 11 and new gradle

### PR DESCRIPTION
**Pull request by ThadHouse**
Note this is broken at runtime, I think Austin had a fix somewhere.

### CODE REVIEW

**Comment by ThadHouse** [comment]
I removed extension support for now until the classpath loading changes can be figured out. That way we can have at least something for beta.

File: src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java [hunk 1]
```diff
@@ -69,7 +69,8 @@ public void run() {
         1000);
 
     // Search the filesystem for extensions (49%)
-    FileSniffer.findExtensions(monitor, 0, 490);
+    //FileSniffer.findExtensions(monitor, 0, 490);
+    // Extensions disabled until classloading can be fixed
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java]: You should probably add an issue and link to it from here.

Comment (ThadHouse) [reply src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java]: Done

File: src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package edu.wpi.first.smartdashboard;
 
-import edu.wpi.first.smartdashboard.extensions.FileSniffer;
+//import edu.wpi.first.smartdashboard.extensions.FileSniffer;
```
Comment (JLLeitschuh) [inline src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java]: Probably better to remove this import instead of commenting it out.

Comment (ThadHouse) [reply src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java]: Fixed

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (JLLeitschuh) [inline .vscode/settings.json]: 😕

Comment (ThadHouse) [reply .vscode/settings.json]: Added to git ignore.

**Pull request merged**